src/api/routes/meetings.ts

```diff
@@ -113,9 +115,103 @@ const decline = procedure
     商量解决方案。`, baseUrl);
 });
 
+function validateMeetingSlot(
+  values: { tmUserId?: string; meetingId: string; meetingLink: string },
+  isCreating: boolean
+): void {
+  if (isCreating && !values.tmUserId?.trim()) {
+    throw new TRPCError({
+      code: 'BAD_REQUEST',
+      message: '腾讯会议用户ID不能为空',
+    });
+  }
+  
+  if (!values.meetingId.trim()) {
+    throw new TRPCError({
+      code: 'BAD_REQUEST', 
+      message: '会议ID不能为空',
+    });
+  }
+  
+  if (!values.meetingLink.trim()) {
+    throw new TRPCError({
+      code: 'BAD_REQUEST',
+      message: '会议链接不能为空',
+    });
+  }
+  
+  try {
+    new URL(values.meetingLink);
+  } catch {
+    throw new TRPCError({
+      code: 'BAD_REQUEST',
+      message: '会议链接格式不正确',
+    });
```

Comment:
use `generalBadRequestError` instead.

also, do not trim here. the caller should trim